playbooks/nfs.yml

```diff
@@ -0,0 +1,107 @@
+---
+# Copyright 2024 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Deploy an NFS server
+  become: true
+  hosts: "{{ groups[cifmw_nfs_target | default('computes')][0] | default([]) }}"
+  pre_tasks:
+    - name: End play early if no NFS is needed
+      when:
+        - not cifmw_edpm_deploy_nfs | default('false') | bool
```

Comment:
hmm and we should document that new top-level parameter I think (sorry for not mentioning it earlier - too many things on my plate).
Somewhere in https://github.com/openstack-k8s-operators/ci-framework/blob/main/docs/source/usage/01_usage.md

Reply:
True, I introduce a few parameters in this change, I'll add them all to the docs and update once the current CI run finishes, thanks for the link!